### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+
+[Full Changelog](https://github.com/voxpupuli/puppet-sftp_jail/compare/v1.1.0...v2.0.0)
+
+**Closed issues:**
+
+- concat::fragment error when initing jail  [\#26](https://github.com/voxpupuli/puppet-sftp_jail/issues/26)
+
+**Merged pull requests:**
+
+- replace validate\_\* with datatypes in init [\#30](https://github.com/voxpupuli/puppet-sftp_jail/pull/30) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (alexjfisher) [inline CHANGELOG.md]: Do you want to mark this as the breaking change?

Comment (bastelfreak) [reply CHANGELOG.md]: updated

**Comment by bastelfreak** [comment]
added labels to the other PRs and regenrated the changelog

**Comment by alexjfisher** [comment]
Perfect

**Pull request merged**